mne/preprocessing/ica.py

```diff
@@ -3153,17 +3153,33 @@ def _band_pass_filter(inst, sources, target, l_freq, h_freq, verbose=None):
 
 def _find_max_corrs(all_maps, target, threshold):
     """Compute correlations between template and target components."""
+    # Following Fig.2 from:
+    # https://www.sciencedirect.com/science/article/abs/pii/S1388245709002338
+
+    # > ... inverse weights (i.e., IC maps) from a selected template IC are
+    # > correlated with all ICs from all datasets ...
     all_corrs = [compute_corr(target, subj.T) for subj in all_maps]
     abs_corrs = [np.abs(a) for a in all_corrs]
```

Comment:
... isn't this just `np.abs(all_corrs)`?

This was back when I was just learning Python :(

Reply:
... I think only if `all_corrs` is not ragged, which it could be if there are different numbers of maps per subject. Not sure if this is ever the case or not, but I think I'd rather not mess with it.